frontend/src/views/SettingsView.vue

```diff
@@ -1,7 +1,291 @@
+  },
+  methods: {
+    ...mapActions(useUsersStore, ['getUsersPermissionsFacilities']),
+
+    /**
+     * Get the users, permissions, and facilities for an organization
+     */
+    async getUsersFacilities(organizationId) {
+      await this.getUsersPermissionsFacilities(this.userInfo.organizationId)
+    },
+
+    /**
+     * Toggle the facility input filter
+     */
+    toggleShowFilter() {
+      if (this.showFilterInput) {
+        this.showFilterInput = false
+        this.facilityNameFilter = ''
+      } else {
+        this.showFilterInput = true
+      }
+    },
+
+    /**
+     * Toggle expanded row
+     */
+    toggleExpand(item) {
+      const index = this.expanded.indexOf(item.contactId);
+      if (index > -1) {
+        // Collapse if already expanded
+        this.expanded.splice(index, 1);
+      } else {
+        // Expand the clicked row
+        this.expanded = [item.contactId];
+      }
+    },
+
+    // TODO: Hard coding of role names in this method is temporary. Will be replaced with endpoint call to get role names.
+    /**
+     * Get the role description for role code(s)
+     */
+    getRoleDescriptions(roles) {
+      const roleDescriptions = []
+      let rolesArray = roles.split(',').map(Number);
+      rolesArray.forEach(role => {
+        switch (role) {
+          case this.ROLES.ADMIN:
+            roleDescriptions.push('Admin')
+            break
+          case this.ROLES.ACCOUNT_MANAGEMENT:
+            roleDescriptions.push('Account Management')
+            break
+          case this.ROLES.FINANCIAL:
+            roleDescriptions.push('Financial')
+            break
+          case this.ROLES.REPORTING:
+            roleDescriptions.push('Reporting')
+            break
+        }
+      })
+      return roleDescriptions.join(', ')
+    },
+
+    /**
+     * Get the status description for a status code
+     */
+    getStatusDescription(statusCode) {
+      switch (statusCode) {
+        case CRM_STATE_CODES.ACTIVE:
+          return 'Active'
+        case CRM_STATE_CODES.INACTIVE:
+          return 'Inactive'
+      }
+    },
+  },
+}
+</script>
+
+<style scoped>
+.sub-header {
+  display: flex;
+  color: #003366;
+  font-weight: 600;
+  font-size: 1.3em;
+}
+
+.table-facility {
+  border: 1px solid #D9D9D9;
+}
+
+.table-header-facility {
+  color: black !important;
+  font-weight: bold !important;
+  background-color: #D9D9D9;
+  font-size: .90em;
+  height: 23px !important;
+}
+
+.table-header-facility:hover {
+  cursor: pointer;
+}
+
+.sub-table-header {
+  color: #545454;
+  font-weight: 600 !important;
+}
+
+td.table-item-facility {
+  height: 23px !important;
+  font-size: .90em !important;
+}
+
+.interactive-text,
+.interactive-text-bold {
+  color: #6699cc;
+  cursor: pointer;
+}
+
+.interactive-text {
+  font-weight: 500;
+}
+
+.interactive-text-bold {
+  font-weight: bold;
+}
+
+.interactive-text:hover,
+.interactive-text-bold:hover {
+  text-decoration: underline;
+}
+
+.no-border-bottom {
```

Comment:
There are vuetify classes for this: pb-0 maybe?

Reply:
pb-0 is padding-bottom, there is no vuetify 3 class name for border-bottom style control. Attempting to put on tr and take off td's also no effect.